senza/templates/postgresapp.py

```diff
@@ -245,6 +263,29 @@ def gather_user_variables(variables, region, account_info):
            default="noatime,nodiratime,nobarrier")
     prompt(variables, "scalyr_account_key", "Account key for your scalyr account", "")
 
+    prompt(variables, 'pgpassword_superuser', "Password for PostgreSQL superuser [random]", show_default=False,
+           default=generate_random_password, hide_input=True, confirmation_prompt=True)
+    prompt(variables, 'pgpassword_standby', "Password for PostgreSQL user standby [random]", show_default=False,
+           default=generate_random_password, hide_input=True, confirmation_prompt=True)
+    prompt(variables, 'pgpassword_admin', "Password for PostgreSQL user admin", show_default=True,
+           default='admin', hide_input=True, confirmation_prompt=True)
+
+    variables['kms_arn'] = None
+    if click.confirm('Do you wish to encrypt these passwords using kms?', default=False):
+        kms_keys = [k for k in list_kms_keys(region, True) if k['Description'] !=
+                    'Default master key that protects my EBS volumes when no other key is defined']
```

Comment:
Not very, I was hoping to be able to use the key alias (more reliable), but it's not part of the description of the key given by boto3. Will investigate further.